.env.local

```diff
@@ -0,0 +1,2 @@
+EXPO_PUBLIC_SUPABASE_URL=https://jppmwghkztznzhvmwimr.supabase.co
+EXPO_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImpwcG13Z2hrenR6bnpodm13aW1yIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MjE4NTQzNzAsImV4cCI6MjAzNzQzMDM3MH0.yF4d9twWCn205niXGspjMOAcDPFPYR4gubBNmq8-A4E
```

Comment:
oopsies! You shouldn't be committing this file. Check your git ignore, then remove this file from this PR